src/packet.rs

```diff
@@ -195,6 +266,11 @@ impl<'a> MspPacket<'a> {
 		6 + self.data.len()
 	}
 
+	/// Number of bytes that this packet requires to be packed
+	pub fn packet_size_bytes_v2(&self) -> usize {
+		 9 + self.data.len()
+	}
+
```

Comment:
This is rather unfulfilling. I would prefer to have the MspVersion enum in use to determine the actual size of package. However, the MSP version is currently not actually a part of a package...

Reply:
I agree, maybe create something like this https://github.com/magiclen/crc-any/blob/b3ab9f45eee99d47534707703e442246206ef105/src/lib.rs#L181 for MspPacket, but i rather keep this change out of this PR and create different one in the future